app/api/v1/cards_api.rb

```diff
@@ -0,0 +1,45 @@
+module V1
+  class Cards_Api < Grape::API
+    HAND_NAME = ["ハイカード","ワンペア","ツーペア","スリー・オブ・ア・カインド","ストレート","フラッシュ","フルハウス","フォー・オブ・ア・カインド","ストレートフラッシュ"]
+
+    resource "card"  do
+      params do
+        requires :cards , type: Array
+      end
+      post"/judge"  do
+        cards =  params[:cards]
+        all_hand = []
+        results = []
+        cards.each do |card|
+           @hands = Cards.new(card)
+           @hands.change_card_to_numbers_and_suits
```

Comment:
If you use "change" in a method name, please show change from where to where, or using concrete word is better.